pkg/operator/starter.go

```diff
@@ -243,7 +243,12 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 	)
 
 	clusterMemberController := clustermembercontroller.NewClusterMemberController(
+		masterNodeLabelSelector,
+		masterMachineLabelSelector,
+		ceohelpers.NewMachineAPI(masterMachineInformer, machinelistersv1beta1.NewMachineLister(masterMachineInformer.GetIndexer()), masterMachineLabelSelector),
```

Comment:
multiple controllers need `Machine API`, I would factor out and pass a singleton.